packages/fixie/src/auth.ts

```diff
@@ -76,7 +76,7 @@ export async function Authenticate({
     // No key available. Need to punt.
     return null;
   }
-  const client = FixieClient.Create(useApiUrl, useApiKey);
+  const client = new FixieClient({ apiKey: useApiKey, url: useApiUrl });
```

Comment:
Can you please add a docstring for the constructor explaining the behavior with respect to how the key and URL are chosen? (The comment above on lines 67-71 is important)

Reply:
I don't totally follow -- PTAL at the latest version and let me know if I should still change it